config/crd/bases/argoproj.io_applications.yaml

```diff
@@ -12,519 +12,176 @@ spec:
     listKind: ApplicationList
     plural: applications
     shortNames:
-    - app
-    - apps
+      - app
```

Comment:
These are formatting changes. May be done via auto formatting. These may not be required.
https://github.com/argoproj/argo-cd/blob/v2.11.9/manifests/crds/application-crd.yaml